opendevin/runtime/docker/ssh_box.py

```diff
@@ -221,22 +311,31 @@ def read_logs(self, id) -> str:
         bg_cmd = self.background_commands[id]
         return bg_cmd.read_logs()
 
-    def execute(self, cmd: str) -> tuple[int, str]:
-        cmd = cmd.strip()
-        # use self.ssh
+    def execute(self, cmd: str) -> Tuple[int, str]:
+        commands = split_bash_commands(cmd)
+        if len(commands) > 1:
+            all_output = ''
+            for command in commands:
+                exit_code, output = self.execute(command)
+                if all_output:
+                    all_output += '\r\n'
```

Comment:
Hello @rbren 
since we don't support Windows, why is CR+LF used here (still in main today)?
I think this could cause issues on MacOS?